hyracks-fullstack/hyracks/hyracks-dataflow-common/src/main/java/org/apache/hyracks/dataflow/common/data/partition/range/RangeMap.java

```diff
@@ -20,7 +20,8 @@
 
 import java.io.Serializable;
 import java.util.Arrays;
-import java.util.Objects;
```

Comment:
I think my editor might have removed this line automatically before I copied the hashCode and equals methods in, but eclipse hasn't complained that it's missing